libparsec/crates/platform_mountpoint/build.rs

```diff
@@ -0,0 +1,6 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+fn main() {
+    #[cfg(target_os = "windows")]
```

Comment:
Add comment:
```rust
// WinFSP is provided as a DLL that our main binary must loaded.
// By default Windows look for DLL in 1) the executable directory and 2) the Windows system folders.
// However this doesn't work for WinFSP given it is distributed as a separate program
// (and hence end up install in an arbitrary directory we must retrieve at the runtime
// by querying the Windows Registry) .
// Long story short, we are here here informing the linker the WinFSP DLL must be lazy
// loaded ("delayload" option in MSVC) so that we will have time to first configure the
// lookup directory.
```

(this comment is pretty sweet, you should also put it in https://github.com/Scille/winfsp_wrs/blob/main/winfsp_build/src/lib.rs ^^)

Reply:
Done